controllers/backup_controller.go

```diff
@@ -197,7 +197,7 @@ func (r *FoundationDBBackupReconciler) updateOrApply(ctx context.Context, backup
 			Status: backup.Status,
 		}
 
-		return r.Status().Patch(ctx, patch, client.Apply, client.FieldOwner("fdb-operator"), client.ForceOwnership)
+		return r.Status().Patch(ctx, patch, client.Apply, client.FieldOwner("fdb-operator")) //, client.ForceOwnership)
```

Comment:
I am not sure how this change is related to upgrading chaos-mesh.

Reply:
Those changes are related to the new controller runtime, there were some changes that made the `client.ForceOwnership` not a correct argument for sub resources.